android/vcpkg_config_libs/vcpkg.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "boinc-libs",
```

Comment:
Hm, my bad. This should be still 'libboinc'

Reply:
This is still need to be fixed

Reply:
This is not the same as libboinc cmake.
This is libs component of boinc, as the name suggest.